src/applications/disability-benefits/all-claims/content/instructionalPart2.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const instructionalPart2Description = (
```

Comment:
Same question as above, should it be a default export?

Reply:
changed. Thank you!